tools/sdk-generation-pipeline/Dockerfile

```diff
@@ -0,0 +1,62 @@
+FROM ubuntu:20.04
+
+RUN apt-get update -y &&  apt upgrade -y && apt install curl -y
+RUN apt install build-essential -y
+
+# install java
+ENV JAVA_HOME /usr/java/openjdk-17
```

Comment:
We will need to eventually come up with a plan to keep these language specific tools in-sync with what we are putting into this docker. One possible solution would be to somehow share this data or rely more on the init scripts instead of baking it into the image.

Reply:
Yes, I agree. I planned to depends on the init script or add a new script in sdk repository to help install the development environment. But I prefer to hardcode it in the docker image v1.0 because it doesn't need any commits to sdk repository.

Reply:
OK if that is the plan lets please be sure to have a tracking issue to go back and fix this as it will be very easy to forget to fix that.